client/src/utils/formatters.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
@dzimmerman902 When you get a chance, can you confirm that this is the right place and naming for a shared function like this?

Reply:
@seankwilliams Yup! That is exactly where it needs to go!

Reply:
thanks!